src/LogicApp/LogicApp/Cmdlets/IntegrationAccount/GetAzureIntegrationAccountMapCommand.cs

```diff
@@ -45,6 +44,11 @@ public class GetAzureIntegrationAccountMapCommand : LogicAppBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string MapName { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "The integration account map type.")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet("Xslt", "Xslt20", "Xslt30", "Liquid", IgnoreCase = true)]
```

Comment:
@refortie rather than using `ValidateSet`, please use the `PSArgumentCompleter` attribute -- they enable the same tab completion for users, but `PSArgumentCompleter` allows outside values if the user decides to provide one and the server is able to accept it in the future.

Reply:
Good to know. Will change